wisecondorX/wisetools.py

```diff
@@ -480,7 +480,7 @@ def generate_txt_output(args, json_out):
 
     segments_file.close()
 
-    statistics_file = open(args.outid + "_statistics.txt", "w")
+    statistics_file = open(args.outid + "_chr_statistics.txt", "w")
```

Comment:
is there a reason the output filename changes? This will cause errors downstream.

Reply:
I believe this name makes more sense, since all output files contain statistics, yet, it doesn't really matter